tools/scoring/score/dimensions/result.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Outputs of the configuration scoring tool."""
+from dataclasses import dataclass
+
+
+@dataclass
+class Result:
+  """
+    Container for floating-point results which
+    quantify success in each dimension
+
+    Attributes:
+      correct_virtual: Number of correct attempts within virtual devices
+      correct_reporting: Number of correct attempts within reporting devices
+      correct_ceiling_virtual: Number of correct attempts possible
+        within virtual devices
+      correct_ceiling_reporting: Number of correct attempts possible
+        within reporting devices
+      incorrect_virtual: Number of incorrect attempts within virtual devices
+      incorrect_reporting: Number of incorrect attempts within reporting devices
+
+    Properties:
+      result_composite: Calculated result for all devices
```

Comment:
total_result?

Reply:
More intuitive verbiage might be helpful but total implies a sum, I think…

Reply:
I see, sgtm